rocky/reports/templates/partials/report_ooi_list.html

```diff
@@ -78,10 +81,10 @@ <h2>
                         <th>{% translate "Clearance type" %}</th>
                     </thead>
                     <tbody>
-                        {% for object in object_list %}
+                        {% for object in ooi_list %}
                             <tr>
                                 <td>
-                                    <input type="checkbox" id="submit-form" name="ooi" value={{ object.primary_key }} {% if object.primary_key in selected_oois or "all" in request.GET.ooi %}checked{% endif %} {% if "all" in request.GET.ooi %}disabled{% endif %}>
+                                    <input type="checkbox" name="ooi" value={{ object.primary_key }} {% if object.primary_key in selected_oois or "all" in request.GET.ooi %}checked{% endif %} {% if "all" in request.GET.ooi %}disabled{% endif %}>
```

Comment:
The value attribute should be enclosed with double quotes (and preferable encoded?). It's unsafe otherwise, since the primary key is basically user input.

Reply:
I forgot to put the double quotes.